src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModalContainer.tsx

```diff
@@ -46,9 +46,9 @@ const ActivateWorkPackageModalContainer: React.FC<ActivateWorkPackageModalContai
         submitterId: auth.user?.userId,
         wbsNum,
         type: ChangeRequestType.Activation,
-        projectLeadId,
-        projectManagerId,
-        startDate,
+        projectLeadId: projectLeadId || 0,
```

Comment:
Instead of doing `projectLeadId || 0` in the case that projectLeadId is undefined, use an if statement like above to check if `projectLeadId` is defined and throw an error if not